chain/network/src/private_actix.rs

```diff
@@ -0,0 +1,178 @@
+/// This file is contains all types used for communication between `Actors` within this crate.
+/// They are not meant to be used outside.
```

Comment:
And also they are not persisted anywhere, right ?

Reply:
@mm-near No. The is no way to persist them, because they don't derive `borsh` nor `serde`.